solr/core/src/test/org/apache/solr/filestore/TestDistribPackageStore.java

```diff
@@ -225,6 +229,11 @@ public String toString() {
     }
   }
 
+  public static <T extends NavigableObject> T assertResponseValues(
+      Callable<T> callable, Map<String, Object> vals) throws Exception {
+    return assertResponseValues(1, callable, vals);
```

Comment:
Noting that it seems that (at least some) of the callers used `10` repeats before but now the default is `1` here.

Reply:
I looked through that and I think it was left over from before SOLR-14749 when we added proper concurrency controls. In this current case, we really shouldn't need to be retrying anymore.